packages/hash/api/src/model/knowledge/entity.model.ts

```diff
@@ -334,7 +334,7 @@ export default class {
     params: {
       entityId: EntityId;
     },
-  ): Promise<EntityModel> {
+  ): Promise<EntityModel | LinkEntityModel> {
```

Comment:
Yeah, agree it doesn't feel ideal either but since we're planning on reworking the model class approach soon anyways I'm not sure it's worth revising if it works.